plugins/ctx.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ctx
+spec:
+  homepage: https://github.com/ahmetb/kubectx
+  shortDescription: Switch between clusters in your kubeconfig
```

I would say
```suggestion
  shortDescription: Switch between contexts in your kubeconfig
```
because users may define different contexts for the same cluster.

---

Also, it would be nice to squeeze a reference to `kubectx` in there, so that existing `kubectx` users know that they get same tool here _plus_ automatic updates via krew. For example

```suggestion
  shortDescription: Switch between contexts (same as kubectx)
```